src/serializers/decoders/current/v12/decode_graph_schema.c

```diff
@@ -130,8 +130,8 @@ static Schema *_RdbLoadSchema
 
 	if(s) {
 		// no entities are expected to be in the graph in this point in time
-		if(s->index) Index_Construct(s->index, gc->g);
-		if(s->fulltextIdx) Index_Construct(s->fulltextIdx, gc->g);
+		if(s->index) Index_Populate(s->index, gc->g);
```

Comment:
in no entities why we need to call Index_Populate ?

Reply:
changed to:
```c
// construct index structure
if(!already_loaded) {
    Index_ConstructStructure(idx);
}
```